CHANGELOG/CHANGELOG-4.0.md

```diff
@@ -0,0 +1,261 @@
+# Release notes for v4.0.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v3.0.0
+
+## Breaking Changes
+
+- Update to client/v4 ([#451](https://github.com/kubernetes-csi/external-snapshotter/pull/451), [@xing-yang](https://github.com/xing-yang))
+
+- Move Snapshot APIs to V1. ([#419](https://github.com/kubernetes-csi/external-snapshotter/pull/419), [@xing-yang](https://github.com/xing-yang))
+
+- Update snapshot-controller, CSI snapshotter sidecar, and validation webhook based on v1 apis. ([#425](https://github.com/kubernetes-csi/external-snapshotter/pull/425), [@xing-yang](https://github.com/xing-yang))
```

Comment:
Note that this requires the v1 CRD to be installed in the cluster.

Reply:
Added a note there.